tests/guest-tools/unix/test_guest_tools_unix.py

```diff
@@ -34,9 +36,7 @@ def _check_os_info(self, vm, vm_distro):
     def test_install(self, running_vm, state):
         vm = running_vm
 
-        # skip test for windows and some unixes
-        if vm.is_windows:
-            pytest.skip('Test module only valid for Unix VMs')
+        # skip test for some unixes
```

Comment:
The comment is no longer needed.

Reply:
We still skip alpine, that's why I left it.